docs/rfcs/0007-deployment-chain.md

```diff
@@ -0,0 +1,275 @@
+- Start Date: 2021-11-03
+- Target Version: 0.22.0
+
+# Summary
+This RFC introduces a new way to enable users to connect multiple applications to a complex deployment pipeline. It can bring to users the ability to roll out to multiple clusters gradually or promote across environments.
+
+# Motivation
+With the current supporting deployment, our users have to define their application as one of the supporting application's kinds (K8s, Terraform, CloudRun, Lambda, ECS), they can still define multiple PipeCD applications as parts of their application and trigger them one by one, but it's a bit difficult for them to control the deployment chain smoothly.
+
+![image](assets/deployment-chain-kind.png)
+Use-case <1>: chain of deployments contains multiple application cross kinds
+
+Another typical use-case is when users want to build up some kind of chain such as deploy application to the development environment, then if it's done successfully, deploy that application to the staging and then the production environment.
+
```

Comment:
How about following the layout below to be easier to read?

```
Use-case <1> ...
Image 1
Description text for use-case 1

Use-case <2> ...
Image 2
Description text for use-case 2


Use-case <3> ...
Image 3
Description text for use-case 3
```

Reply:
Adopted 🙆‍♀️